app/views/clients/_housing_requirements.haml

```diff
@@ -7,7 +7,9 @@
       %span.c-detailed-list__label Housing Release
       %span.c-detailed-list__value
         = _(@client.housing_release_status.presence) || 'Not Signed'
-
+    %li.c-detailed-list__item
+      %span.c-detailed-list__label Most recent assessment completion date
+      %span.c-detailed-list__value= @client.entry_date.try(:strftime, I18n.t('date.formats.default'))
```

Comment:
Another update based on QA feedback, to add the entry date tie-breaker to the client page. I added it here:

<img width="1307" alt="Screen Shot 2022-08-30 at 9 42 23 AM" src="https://user-images.githubusercontent.com/5333982/187453380-97d10593-bbb4-4b31-b05d-d4bbccc563ac.png">

Reply:
isn't `entry_date` already on that page?

Reply:
huh, guess not.